### ISSUE TITLE
AccessibilityInfo not found after installing module

### ISSUE THREAD

**Issue by moeinrahimi** [joinspontaneous/react-native-loading-spinner-overlay#53]
hi
just after installing this moulde I'm getting this error
```
bundling: Error: Cannot find module 'AccessibilityInfo' (While processing preset: "C:\\Users\\moein\\Desktop\\wishList\\node_modules\\react-native\\Libraries\\react-native\\react-native-implementation.js")
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.get AccessibilityInfo [as AccessibilityInfo] (C:\Users\moein\Desktop\wishList\node_modules\react-native\Libraries\react-native\react-native-implementation.js:19:36)
    at C:\Users\moein\Desktop\wishList\node_modules\lodash\_baseClone.js:145:23
    at arrayEach (C:\Users\moein\Desktop\wishList\node_modules\lodash\_arrayEach.js:15:9)
    at baseClone (C:\Users\moein\Desktop\wishList\node_modules\lodash\_baseClone.js:142:3)
    at cloneDeepWith (C:\Users\moein\Desktop\wishList\node_modules\lodash\cloneDeepWith.js:37:10)
TransformError: C:\Users\moein\Desktop\wishList\index.android.js: Cannot find module 'AccessibilityInfo' (While processing preset: "C:\\Users\\moein\\Desktop\\wishList\\node_modules\\react-native\\Libraries\\react-native\\re
act-native-implementation.js")
Bundling `index.android.js`  0.0(0/1), failed.
```

**Comment by zptlwin** [comment]
Hello moeinrahimi,

How do u solve this error?
I also got that error.

Thanks,

**Comment by moeinrahimi** [comment]
I didn't find any proper way to solve this issue, at last I just created a new react-native app and copied my project folders and files in it, I don't know why this happens but i think it's related to using yarn and npm at once in one project

**Comment by michaelstokes** [comment]
Regards this, I just ran `npm install react-native --save` and it fixed it.
